signup.html

```diff
@@ -24,20 +24,20 @@
         <div class="input-container pw-container">
           <label for="pw">비밀번호</label>
           <input id="pw" type="password" placeholder="비밀번호를 입력해주세요">
-          <img class="btn-off" src="./asset/icon/btn_visibility_off.png">
+          <img class="btn-off" src="./assets/icons/btn_visibility_off.png">
         </div>
         <div class="input-container pw-container">
           <label for="pw">비밀번호 확인</label>
           <input id="pw" type="password" placeholder="비밀번호를 다시 입력해주세요">
-          <img class="btn-off" src="./asset/icon/btn_visibility_off.png">
+          <img class="btn-off" src="./assets/icons/btn_visibility_off.png">
         </div>
         <button class=button type="submit">회원가입</button>
       </form>
       <div class="login-messenger">
         <p>간편 로그인 하기</p>
         <div class="messengers">
-          <a href="https://www.google.com/"><img src="asset\icon\ic_google.png"></a>
-          <a href="https://www.kakaocorp.com/page/"><img src="asset/icon/ic_kakaotalk.png"></a>
+          <a href="https://www.google.com/"><img src="assets\icons\ic_google.png"></a>
+          <a href="https://www.kakaocorp.com/page/"><img src="assets/icons/ic_kakaotalk.png"></a>
```

Comment:
```suggestion
          <a href="https://www.google.com/">
              <img src="./assets\icons\ic_google.png">
          </a>
          <a href="https://www.kakaocorp.com/page/">
              <img src="./assets/icons/ic_kakaotalk.png">
          </a>
```
* 중첩된 태그는 줄바꿈하면 더 잘 보이더라구요:-)